core/server/services/members/utils.js

```diff
@@ -0,0 +1,15 @@
+module.exports.formattedMemberResponse = function formattedMemberResponse(member) {
+    if (!member) {
+        return null;
+    }
+    return {
+        uuid: member.uuid,
+        email: member.email,
+        name: member.name,
+        firstname: member.name && member.name.split(' ')[0],
+        avatar_image: member.avatar_image,
+        subscribed: !!member.subscribed,
+        subscriptions: member.stripe ? member.stripe.subscriptions : [],
+        paid: member.stripe ? member.stripe.subscriptions.length !== 0 : false
+    };
+};
```

Comment:
Since multiple endpoints on members api need to format and send member data, extracted this into a common local util